src/equations/numerical_fluxes.jl

```diff
@@ -221,6 +221,46 @@ See [`FluxLaxFriedrichs`](@ref).
 """
 const flux_lax_friedrichs = FluxLaxFriedrichs()
 
+"""
```

Comment:
```suggestion
@doc raw"""
```
The docs are not rendering nicely, see https://trixi-framework.github.io/Trixi.jl/previews/PR2204/reference-trixi/#Trixi.DissipationLaxFriedrichsEntropyVariables.

Reply:
Thanks! I didn't know that it was possible to see a preview 😎

Reply:
[Hendrik's suggestion](https://github.com/trixi-framework/Trixi.jl/pull/2204#discussion_r1889989923) fixed the docs.

Reply:
Alright. I considered it unnecessary because the issue was resolved. I accepted the `@doc raw` suggestion in https://github.com/trixi-framework/Trixi.jl/pull/2204/commits/c98006ffdab3786df9af9135827ed36c3961b48c...